src/js/services/limit_order/limit_order.js

```diff
@@ -422,3 +425,28 @@ export function updateFavoritePairs(base, quote, to_fav){
     })
   })
 }
+
+export function getTokenPrice(symbol){
```

Comment:
This func should be removed to services/ethereum/nodeProviders/CachedServerProvider.js

Reply:
Removed